Gems/LmbrCentral/Code/Tests/EditorPolygonPrismShapeComponentTests.cpp

```diff
@@ -171,8 +171,8 @@ namespace LmbrCentral
         const auto screenStart = AzFramework::WorldToScreen(worldStart, m_cameraState);
         const auto screenEnd = AzFramework::WorldToScreen(worldEnd, m_cameraState);
 
-        // small diagonal offset to ensure we interact with the planar manipulator and not one of the linear manipulators
-        const AzFramework::ScreenVector offset(5, -5);
+        // diagonal offset to ensure we interact with the planar manipulator and not one of the linear manipulators
+        const AzFramework::ScreenVector offset(30, -30);
```

Comment:
Could this test fail locally if you have local changes to the settings?

Reply:
Yes that's an excellent shout - I'm going to do that in the follow-up PR I'm working on where the sizes will be easier to expose